plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/CliOptions.java

```diff
@@ -107,14 +107,18 @@ private File getPluginTxt() {
      *
      */
     private File getPluginDir() {
-        if (pluginDir == null) {
-            System.out.println("No directory to download plugins entered. " +
-                    "Will use default of " + Settings.DEFAULT_PLUGIN_DIR);
-            return Settings.DEFAULT_PLUGIN_DIR;
-        } else {
+        if (pluginDir != null) {
             System.out.println("Plugin download location: " + pluginDir);
             return pluginDir;
+        } else if (!StringUtils.isEmpty(System.getenv("REF")))  {
```

Comment:
ref doesn't mean anything to me

From what I understand it is doing:
```suggestion
        } else if (!StringUtils.isEmpty(System.getenv("PLUGIN_DIR")))  {
```

Possibly you may want to keep the same env for initial compatibility but it shouldn't be the main one in my opinion, and even so I would consider just dropping it